libs/designsystem/slide/src/slides.component.ts

```diff
@@ -1,54 +1,111 @@
-      #ionslides
-      (ionSlideDidChange)="onSlideChanged()"
-      ionSlidesDidLoad="slideTo(0)"
-    >
-      <ion-slide *ngFor="let slide of slides; let i = index">
-        <ng-container
-          *ngTemplateOutlet="slideTemplate; context: { $implicit: slide, index: i }"
-        ></ng-container>
-      </ion-slide>
-    </ion-slides>
-  `,
-  changeDetection: ChangeDetectionStrategy.OnPush,
+  templateUrl: './slides.component.html',
+  styleUrls: ['./slides.component.scss'],
 })
-export class SlidesComponent {
-  @ViewChild('ionslides', { static: false }) ionSlides: IonSlides;
-  @Input() slidesOptions: any;
-  @Input() slides: any[];
-  @Output() selectedSlide = new EventEmitter<any>();
+export class SlidesComponent implements OnInit, AfterViewInit {
+  constructor(private platform: PlatformService) {}
 
+  @ViewChild('swiperContainer') swiperContainer;
   @ContentChild(SlideDirective, { static: true, read: TemplateRef })
   public slideTemplate: TemplateRef<any>;
 
-  onSlideChanged() {
-    this.ionSlides.getActiveIndex().then((selectedIndex) => {
-      this.selectedSlide.emit({
-        slide: this.slides[selectedIndex],
-        index: selectedIndex,
-      });
-    });
+  @Input() slidesOptions: KirbySwiperOptions;
+  @Input() ignorePagePadding: boolean;
+  @Input() title: string;
+  @Input() slides: any[];
+
+  // simpleSlider is a temporary solution to make the slides component backwards compatible with the old design.
+  @Input() simpleSlider = false;
+
+  @Output() selectedSlide = new EventEmitter<SelectedSlide>();
+
+  _paginationId = UniqueIdGenerator.scopedTo('pagination').next();
+  _prevButtonId = UniqueIdGenerator.scopedTo('swiper-button-prev').next();
+  _nextButtonId = UniqueIdGenerator.scopedTo('swiper-button-next').next();
+  _isTouch: boolean;
+
+  ngOnInit() {
+    this._isTouch = this.platform.isTouch();
+  }
+
+  ngAfterViewInit() {
+    const defaultConfig = this.simpleSlider ? this.getSimpleConfig() : this.getDefaultConfig();
+
+    const config = Object.assign(this.slidesOptions, defaultConfig);
+
+    Object.assign(this.swiperContainer.nativeElement, config);
+
+    this.swiperContainer.nativeElement.initialize();
   }
 
   public slideTo(index: number) {
-    this.ionSlides.slideTo(index);
+    this.swiperContainer.nativeElement.swiper.slideTo(index);
+  }
+
+  private getDefaultConfig(): KirbySwiperOptions {
+    return {
+      spaceBetween: 16,
+      centeredSlidesBounds: true,
+      speed: 600,
+      pagination: {
+        el: `.${this._paginationId}`,
+        type: 'bullets',
+      },
+      navigation: {
+        nextEl: `.${this._nextButtonId}`,
+        prevEl: `.${this._prevButtonId}`,
+      },
+      on: {
+        slideChange: (swiper) => {
+          this.selectedSlide.emit({
+            slide: this.slides[swiper.activeIndex],
+            index: swiper.activeIndex,
+          });
+        },
+      },
+    };
+  }
+
+  private getSimpleConfig(): KirbySwiperOptions {
```

Comment:
Consider naming:
```suggestion
  private getSimpleDefaultConfig(): KirbySwiperOptions {
```
OR
```suggestion
  private getLegacyDefaultConfig(): KirbySwiperOptions {
```